src/test/java/org/simulator/stochastic/StochasticTestSuiteTest.java

```diff
@@ -56,110 +56,108 @@
+   * <mi>t</mi>
+   * </mrow>
+   * </msub>
+   * <mo>=</mo>
+   * <msqrt>
+   * <mi>n</mi>
+   * </msqrt>
+   * <mo>&#x2217;<!-- ∗ --></mo>
+   * <mfrac>
+   * <mrow>
+   * <mo stretchy="false">(</mo>
+   * <msub>
+   * <mi>X</mi>
+   * <mrow class="MJX-TeXAtom-ORD">
+   * <mi>t</mi>
+   * </mrow>
+   * </msub>
+   * <mo>&#x2212;<!-- − --></mo>
+   * <mi>m</mi>
+   * <msub>
+   * <mi>u</mi>
+   * <mrow class="MJX-TeXAtom-ORD">
+   * <mi>t</mi>
+   * </mrow>
+   * </msub>
+   * <mo stretchy="false">)</mo>
+   * </mrow>
+   * <mrow>
+   * <mi>s</mi>
+   * <mi>i</mi>
+   * <mi>g</mi>
+   * <mi>m</mi>
+   * <msub>
+   * <mi>a</mi>
+   * <mrow class="MJX-TeXAtom-ORD">
+   * <mi>t</mi>
+   * </mrow>
+   * </msub>
+   * </mrow>
+   * </mfrac>
    * </math>
-   *
+   * <p>
    * This Mean distance function should always fall in (-MEAN_CUTOFF, MEAN_CUTOFF).
-   *
    */
   private static final double MEAN_CUTOFF = 3d;
 
   /**
    * The constant for comparing the standard deviation distances inequality.
-   *
+   * <p>
    * SD distance function
    * <math xmlns="http://www.w3.org/1998/Math/MathML" display="block">
-   *   <msub>
-   *     <mi>Y</mi>
-   *     <mrow class="MJX-TeXAtom-ORD">
-   *       <mi>t</mi>
-   *     </mrow>
-   *   </msub>
-   *   <mo>=</mo>
-   *   <msqrt>
-   *     <mfrac>
-   *       <mi>n</mi>
-   *       <mn>2</mn>
-   *     </mfrac>
-   *   </msqrt>
-   *   <mo>&#x2217;<!-- ∗ --></mo>
-   *   <mo stretchy="false">(</mo>
-   *   <mfrac>
-   *     <msubsup>
-   *       <mi>S</mi>
-   *       <mrow class="MJX-TeXAtom-ORD">
-   *         <mi>t</mi>
-   *       </mrow>
-   *       <mrow class="MJX-TeXAtom-ORD">
-   *         <mn>2</mn>
-   *       </mrow>
-   *     </msubsup>
-   *     <mrow>
-   *       <mi>s</mi>
-   *       <mi>i</mi>
-   *       <mi>g</mi>
-   *       <mi>m</mi>
-   *       <msubsup>
-   *         <mi>a</mi>
-   *         <mrow class="MJX-TeXAtom-ORD">
-   *           <mi>t</mi>
-   *         </mrow>
-   *         <mn>2</mn>
-   *       </msubsup>
-   *     </mrow>
-   *   </mfrac>
-   *   <mo>&#x2212;<!-- − --></mo>
-   *   <mn>1</mn>
-   *   <mo stretchy="false">)</mo>
+   * <msub>
+   * <mi>Y</mi>
+   * <mrow class="MJX-TeXAtom-ORD">
+   * <mi>t</mi>
+   * </mrow>
+   * </msub>
+   * <mo>=</mo>
+   * <msqrt>
+   * <mfrac>
+   * <mi>n</mi>
+   * <mn>2</mn>
+   * </mfrac>
+   * </msqrt>
+   * <mo>&#x2217;<!-- ∗ --></mo>
+   * <mo stretchy="false">(</mo>
+   * <mfrac>
+   * <msubsup>
+   * <mi>S</mi>
+   * <mrow class="MJX-TeXAtom-ORD">
+   * <mi>t</mi>
+   * </mrow>
+   * <mrow class="MJX-TeXAtom-ORD">
+   * <mn>2</mn>
+   * </mrow>
+   * </msubsup>
+   * <mrow>
+   * <mi>s</mi>
+   * <mi>i</mi>
+   * <mi>g</mi>
+   * <mi>m</mi>
+   * <msubsup>
+   * <mi>a</mi>
+   * <mrow class="MJX-TeXAtom-ORD">
+   * <mi>t</mi>
+   * </mrow>
+   * <mn>2</mn>
+   * </msubsup>
+   * </mrow>
+   * </mfrac>
+   * <mo>&#x2212;<!-- − --></mo>
+   * <mn>1</mn>
+   * <mo stretchy="false">)</mo>
    * </math>
```

Comment:
Here in this case the reformatting is not very helpful because this is XML-based code which is now all without indentation. Before it was easier to read.

Reply:
Updating it again to previous one.

Reply:
@draeger, Done in the latest commit.